homeassistant/components/netdata/sensor.py

```diff
@@ -157,9 +225,14 @@ def __init__(self, api):
     async def async_update(self):
         """Get the latest data from the Netdata REST API."""
 
+        original_endpoint = self.api.endpoint
         try:
             await self.api.get_allmetrics()
+            # Overwrite endpoint to receive alarms and later restore it again.
+            self.api.endpoint = "alarms?format=json"
```

Comment:
This should be changed in the library instead and probably provide a separate endpoint.

Reply:
The library is out of our control, we can only suggest that change to the maintainers who are outside the HA community.

Reply:
Please make a pull request to the library. We won't accept this into home assistant.

Reply:
Proposed the change at home-assistant-ecosystem/python-netdata/issues/3

Reply:
Submitted PR home-assistant-ecosystem/python-netdata/pull/5 and changed the code here accordingly